client/src/components/Switch/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import styles from './Switch.module.scss';
+import { uniqueId } from 'lodash';
+import styles from './switch.module.scss'
+class Switch extends React.Component {
+  constructor(props) {
```

Comment:
I am fine with homemade components where needed. If a styled React Material Switch can go here I'd like to keep us to the React Material component kit as much as possible.

Reply:
Changed all the styling to React Material and converted Switch into the functional component